README.md

```diff
@@ -93,6 +93,31 @@ need to build the plugin manually by running the script
 `gymfc/envs/assets/gazebo/plugins/build_plugin.sh`.
 3. Confirm `SetupFile` in `gymfc.ini` is pointing to the correct location.
 
+## Installation of Docker demo (tested on MacOS 10.14.3)
+This repository includes an experimental docker build that demos the usage of GymFC. 
+<p>
+Install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/) and [XQuartz](https://www.xquartz.org/) on your system.
```

Comment:
I looked at your fork and these are rendering correctly, perhaps the HTML tags are causing problems because you never close out the tag. Please remove them so we can stay pure markdown.

Reply:
Still not rendering properly, https://github.com/MichaelManz/gymfc/tree/demo_docker, can you remove all the HTML (i.e., paragraph tags)?

Reply:
Great that fixed it, thank you.